### PR TITLE
CRUD for routes

### PR DESCRIPTION
**Pull request by fab-ryan**
### What does this PR do?

- Integration of CRUD(Create, Read, Update, Delete) with the API where base URL `URL/api/v1/router` and if they are any error will be displayed.

### How should this be tested?

- clone this [Repo](https://github.com/atlp-rwanda/Dominators-Phantom-FE.git)
- checkout to this branch [branch](#ft-integration-crude-operation-for-routes) in other words use this `ft-integration-crude-operation-for-routes`
- run command to install and update packages `npm i ` or `npm install`
- then run command that start serve for production `npm start` and for development `npm run dev`
- user with role of  **Operator** is the one capable of doing many tasks and all other will only _read_ created routes
- on normal user they will use this URL to access page `URL/route` and for user with role of operator, they will use `URL/crud-route`

### What to expect 

- if server successfully connected  all routes will be displayed 
- any error occurs will be pop up  on screen 
- field required are `Origin, Destination, Distance,  Longitude, Latitude, Code`.
- route to be displayed can be altered by changing size of data in table 
- Screen Shot 2022-06-26 at 17 16 59
- display skeleton UI before data fetch 
Screen Shot 2022-06-26 at 17 21 13

### CODE REVIEW

**Review by aimelive — APPROVED** [review]

**Review by GilbertUp — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -8,9 +8,15 @@ lerna-debug.log*
...
 
+<<<<<<< HEAD
+
+yarn.lock
+
+=======
 yarn.lock
 
 
+>>>>>>> 6f4e859bcbb987f7ff882d7d624265c515fd8d08
```
Comment (furebo) [inline .gitignore]: Remove this small conflict in this file

**Review by furebo — CHANGES_REQUESTED** [review]

**Review by Anguandia — CHANGES_REQUESTED** [review]
Good work @fab-ryan just these:
- The PR title is too general, I think you are just dealing with CRUD for routes, not every entity
- Please complete the unfinished rebasing. You still have conflict markers visible in some files!

File: .gitignore (hunk 1 above)
Comment (fab-ryan) [reply .gitignore]: Lets fix this issue with this PR

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
resolve the conflicts

File: .gitignore (hunk 1 above)
Comment (fab-ryan) [reply .gitignore]: I have fixed error

**Review by furebo — APPROVED** [review]

**Pull request merged**